src/runtime/CollectionWrappers/SequenceWrapper.cs

```diff
@@ -0,0 +1,98 @@
+using System;
+using System.Collections.Generic;
+
+namespace Python.Runtime.CollectionWrappers
+{
+    internal class SequenceWrapper<T> : IterableWrapper<T>, ICollection<T>
+    {
+        public SequenceWrapper(PyObject pyObj) : base(pyObj)
+        {
+
+        }
+
+        public int Count
+        {
+            get
+            {
+                return (int)Runtime.PySequence_Size(pyObject.Handle);
```

Comment:
`PySequence_Size` might fail, according to documentation.

Reply:
@lostmsu true, but it doesn't say it throws, so I will throw a c# exception in this case